components/arm/arm.go

```diff
@@ -46,8 +46,10 @@ const SubtypeName = "arm"
 const MTPoob = "cartesian movements are not allowed when arm joints are out of bounds"
 
 var (
-	defaultLinearConstraint  = &motionpb.LinearConstraint{}
-	defaultArmPlannerOptions = &motionpb.Constraints{
+	defaultLinearConstraint = &motionpb.LinearConstraint{}
+
+	// DefaultArmPlannerOptions are supplied for all MoveToPosition calls.
+	DefaultArmPlannerOptions = &motionpb.Constraints{
```

Comment:
Do we want to make this public? We could alternatively hard code this in the UR driver rather than define like this